src/components/simple-color-picker/simple-color/simple-color.test.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+
+import SimpleColor from "./simple-color.component";
+import guid from "../../../__internal__/utils/helpers/guid";
+
+const mockedGuid = "guid-12345";
+jest.mock("../../../__internal__/utils/helpers/guid");
+(guid as jest.MockedFunction<typeof guid>).mockImplementation(() => mockedGuid);
+
+test("has the correct data-component tag", () => {
+  render(<SimpleColor value="#0073C2" data-role="test-example" />);
+
+  // the data tags get bassed to both the wrapping div and the input inside, so just getByTestId errors due to there being more
+  // than one element found. The data-component tag is only on the div so we can't use getByRole - so the only way is to use
+  // getAllByTestId and take the first element
+  expect(screen.getAllByTestId("test-example")[0]).toHaveAttribute(
```

Comment:
**suggestion(non-blocking)**: what do you think about updating `SimpleColor` so the data tags are unique?:

```tsx
<StyledSimpleColor {...tagComponent("simple-color", props)}>
  <StyledSimpleColorInput
    data-element="simple-color-input"
    data-role="simple-color-input"
  />
  {/** ... */}
</StyledSimpleColor>;
```

or removing the tags on `StyledSimpleColorInput`, given you can target the input in RTL like so?:

```tsx
screen.getByRole("radio")
```

Appreciate this would be a breaking change though, since it could impact consumers's UI tests, so happy if you would prefer leaving this for now

Reply:
yes, I don't think this should be done for now. Happy to raise a ticket to maybe do later.